README.md

```diff
@@ -1 +1,194 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Blog App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Expense-App <a name="about-project"></a>
+
+**Expense App** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Initialize your project with Git**
+- **Make sure that your project has Postgres database set up**
+- **Build your project schema**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Video Demo](https://drive.google.com/file/d/1sRQkZAFl4kne2_sGOW_oKQZNynU81oUv/view?usp=sharing)
```

Comment:
nice job with your video demo, your presentation looks good👍🏼. As a requirement, kindly do well to deploy your project, check that everything works fine, and add the link to the live version of the project on your readme file😉😉.


<img width="461" alt="image" src="https://github.com/over-geek/budget-app/assets/64643995/1e83ee97-005b-4a2f-af97-cf87ffda4295">